src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/sourcelink.html

```diff
@@ -0,0 +1,7 @@
+{% if show_source and  has_source and sourcename %}
+<div class="tocsection sourcelink">
+    <a href="{{ pathto('_sources/' + sourcename, true)|e }}">
```

Comment:
It's appending a `.txt` to the source file (so it ends in `.rst.txt`)...I can't tell if that's something happening in this template, or a sphinx configuration thing though

Reply:
https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_sourcelink_suffix that's a Sphinx parameter. we could change it for our doc